pkgs/development/python-modules/spyder/default.nix

```diff
@@ -47,6 +48,7 @@ buildPythonPackage rec {
     mkdir -p $out/share/icons
     cp spyder/images/spyder.svg $out/share/icons
     cp -r $desktopItem/share/applications/ $out/share
+    wrapProgram $out/bin/spyder* --prefix PYTHONPATH : "$PYTHONPATH"
```

Comment:
why?

Reply:
It's because Spyder spawns new python processes which otherwise don't find the needed libraries.  I've added a comment in the code as well.